gas-oracle/Dockerfile

```diff
@@ -2,7 +2,17 @@ FROM golang:1.18.0-alpine3.15 as builder
 
 RUN apk add --no-cache make gcc musl-dev linux-headers git jq bash
 
-COPY ./go/gas-oracle /gas-oracle
+COPY ./batch-submitter /batch-submitter
+COPY ./bss-core /bss-core
+COPY ./gas-oracle /gas-oracle
+COPY ./indexer /indexer
+COPY ./l2geth /l2geth
+COPY ./l2geth-exporter /l2geth-exporter
+COPY ./op-exporter /op-exporter
+COPY ./proxyd /proxyd
+COPY ./teleportr /teleportr
+COPY ./go.work /go.work
```

Comment:
We shouldn't need to copy all into it. We can just build without the `go.work`. Or use a different `go.work` without all the dependencies we don't need.